### DISCUSSION TITLE
viewing steam broadcasts

### DISCUSSION THREAD

**Discussion by wollframium** [chr233/ASFEnhance#82] [功能建议 / Ideas]
now for viewing 10-20 minutes they give backgrounds and some other stuff, can I add a multi-screen in asf?

**Comment by wollframium** [comment]
https://store.steampowered.com/developer/dovetailgames/sale/festivalofrail2023#
За ежедневный просмотр Вам дадут:
2 февраля - 3 фона для Train Sim World
3 февраля - 3 эмоции для Train Sim World
4 февраля - 3 фоновых изображения для Train Simulator Classic
5 февраля - 3 эмоции для Train Simulator Classic

**Comment by SteelLord2547** [comment]
https://steamcommunity.com/broadcast/watch/76561199143003845
watching 15 minutes and you'll get drop
 23 Steam Avatars, Backgrounds and Emotes

**Reply by SteelLord2547** [reply]
it would be nice to add a script so that bots can view

**Reply by molecul** [reply]
Does the bot have to own this game to get the items? I tried to repeat your case, but even after watching 20 minutes, I did not receive any items..

**Reply by wollframium** [reply]
возможно уже прошел ивент?

**Comment by wollframium** [comment]
you don't need to play the video! it is enough just to be on the page

**Reply by SteelLord2547** [reply]
Как сказать ботам сидеть там 15 минут, просто посещение страницы не даст результатов. В ручную, да я охренею

Или я что то туплю? есть команда?

**Reply by wollframium** [reply]
нет
